doc/code/qml_qinfo.rst

```diff
@@ -4,6 +4,9 @@ qml.qinfo
 Overview
 --------
 
+.. warning:: 
+    The `qinfo` module is deprecated and scheduled to be removed in v0.40.
```

Comment:
It is potentially worth adding some information about the corresponding functions in `qml.math` and the measurements here to redirect users.